pkg/workloads/cortex/serve/init/bootloader.sh

```diff
@@ -140,7 +172,7 @@ if [ "$CORTEX_KIND" = "RealtimeAPI" ]; then
 
 # prepare batch otherwise
 else
-    create_s6_service "batch" "cd /mnt/project && $source_env_file_cmd && exec env PYTHONUNBUFFERED=TRUE env PYTHONPATH=$PYTHONPATH:$CORTEX_PYTHON_PATH /opt/conda/envs/env/bin/python /src/cortex/serve/start/batch.py"
+    create_s6_task "batch" "cd /mnt/project && $source_env_file_cmd && PYTHONUNBUFFERED=TRUE PYTHONPATH=$PYTHONPATH:$CORTEX_PYTHON_PATH /opt/conda/envs/env/bin/python /src/cortex/serve/start/batch.py"
```

Comment:
I see we removed the `exec` from here. Does this mean we can also take it out from the `nginx` and `uvicorn` services? If I remember correctly, the services need the exec for some reason. And without the exec in, does the pod stop if the task exits with a non-zero exit code?

Reply:
Without exec the container does exit with the same exit code as the batch process. The correct exit code shows up in k8s.